cms/templates/maintenance/_force_publish_course.html

```diff
@@ -0,0 +1,87 @@
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<div class="wrap-instructor-info studio-view maintenance-form">
+    <form class="form-create" method="post">
+        <input type="hidden" name="csrfmiddlewaretoken" value="${csrf_token}"/>
+        <div class="wrapper-form">
+            <fieldset>
+                <ol class="list-input">
+                    <li class="field text required" id="field-course-name">
+                        <label for="course-id">${_('Course ID')}</label>
+                        <input id="course-id" type="text" name="course-id" aria-describedby="course-id-desc" required
+                               value="${form_data['course_id']}"/>
+                        <div id="course-id-desc" class="tip tip-stacked">${_('e.g. edX+DemoX+Demo_Course')}</div>
+                        <div class="error" aria-live="polite" id="cid-error">
+                        % if error:
+                            ${msg}
+                        % endif
+                        </div>
+                    </li>
+                    <li class="field field-radio">
+                        <div>
+                            % if form_data['is_dry_run']:
+                            <% is_checked = 'checked' %>
+                            % else:
+                            <% is_checked = '' %>
+                            % endif
+                            <input type="checkbox" id="dry-run" name="dry-run" aria-describedby="dry-run-desc" ${is_checked}/>
+                            <label for="dry-run" class="label">
+                            ${_('Dry Run')}
+                            </label>
+                        </div>
+                            <span id="dry-run-desc" class="tip tip-stacked">${_('Trial or practice run the task.')}</span>
+                    </li>
+                </ol>
+            </fieldset>
+        </div>
+        <div class="actions">
+            <button type="submit" class="action action-primary">${_('Force Publish Course')}
+            </button>
+            <a href="${view['url']}" class="action action-secondary action-cancel"
```

Comment:
@adampalay Actually, there are multiple purposes for adding javascript. Just like in the last comment you asked :

> Please disable buttons after you click them to prevent users from clicking multiple times before the requests go through

So, for that I would need Javascript and I have two ways.

One way being adding javascript in `js_extra` block, just like :

```
<%block name="js_extra">
   <script type="text/javascript">
   (function() {
     $("#button_id").click(function(e) {
       // operations to perform
       e.preventDefault();
     })
</script>
</%block>
```

Which is not a good way, we would need to do this for every view, it is not a clean solution.

Second way is that we create backbone view/models for these and do all things from those. 

Second approach is better but it will take good  amount of time but code will look more better and clean.

I think we can create a separate SUST ticket to accomplish that. What do you say ?